torch/csrc/autograd/python_variable.cpp

```diff
@@ -142,16 +156,16 @@ static PyObject *THPVariable_pynew(PyTypeObject *type, PyObject *args, PyObject
 static PyObject* THPVariable_make_subclass(PyObject* _ignored, PyObject* args, PyObject* kwargs) {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
-    "_make_subclass(PyObject* cls, Tensor data, bool require_grad=False)",
+    "_make_subclass(PyObject* cls, Tensor data, bool require_grad=False, bool allow_tensor_metadata_change=True)",
```

Comment:
We need some documentation saying what `allow_tensor_metadata_change` does

Reply:
It was a pre-existing parameter and it would detach the tensor because allow_tensor_metadata_change was set to False. Every tensor would detach when I called make_subclass().